### ISSUE TITLE
STEM: Eligibility summary updating prior to selecting update

### ISSUE THREAD

**Issue by sonntag-adam** [department-of-veterans-affairs/va.gov-team#12185]
When on the summary page of the 10203 form, the eligibility changes prior to selecting update.

Steps to reproduce:
1. Navigate to https://staging.va.gov/education/apply-for-education-benefits/application/10203
2. Fill form all the way out till you get to the summary page https://staging.va.gov/education/apply-for-education-benefits/application/10203/review-and-submit
3. Expand program details
4. View gif below to see it change prior to selecting an update.

Expected behavior: User should click "Update page" before changes to eligibility alert appear and before changes are written to spool file.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/48804582/89575914-77365000-d7fc-11ea-9d91-4d3a4969f123.gif)

**Comment by desireeturner** [comment]
We're waiting on feedback regarding how to handle this from the platform team. cc @emilywaggoner

**Comment by emilywaggoner** [comment]
Consulting with the engineering team on the best way forward - I'll update here shortly @desireeturner!

**Comment by desireeturner** [comment]
@emilywaggoner Any updates on the path forward related to this issue? Thank you!

**Comment by af-manish** [comment]
@naomimarcussen  Can you look at the above and follow-up on how we should move forward?

**Comment by ERRonat0609** [comment]
@mrblanco1 @dan-brooking - I have gone through this form here: https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203

When I got to the review and submit page and expanded the section on Program Details, it appears that the summary info in the gray box has been removed completely from that section. The only place that summary info appears is in the actual form when the user is filling it out.  I've been looking through other tickets that have been mentioned from the original Epic but I cannot find where the change occurred. Perhaps we should have someone in EDU who knows the online forms go through this form and find out if this is truly a non-issue now.

**Comment by mrblanco1** [comment]
Gina to follow up with Joe to verify if the "Summary" has been removed.

**Comment by ERRonat0609** [comment]
Verified with EDU that the actions in teh gray box have been removed. However, this does not fix the problem and furthermore there appears to be a similar issue with other forms, per Cinda Quattrini of EDU. This point to the idea that this may be a platform-level issue. A separate ticket was opened for checking other forms, such as the Feedback tool and 22-1990 that were reported by Cinda. I asked Ryan Thurlwell (OCTO design lead) to review this issue #[34408](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34408).

**Comment by ERRonat0609** [comment]
Closing this ticket to #33408, as this problem is more wide-spread than just a single section of the STEM form as mentioned in the initial description.
